src/core/cubic.c

```diff
@@ -235,24 +236,31 @@ CubicCongestionControlOnCongestionEvent(
     const uint16_t DatagramPayloadLength =
         QuicPathGetDatagramPayloadSize(&Connection->Paths[0]);
     QuicTraceEvent(
-        ConnCongestion,
-        "[conn][%p] Congestion event",
-        Connection);
+        ConnCongestionV2,
+        "[conn][%p] Congestion event: IsEcn=%hu",
+        Connection,
+        Ecn);
+    if (Ecn) {
+        Connection->Stats.Send.EcnCongestionCount++;
```

Comment:
Why inc EcnCongestionCount here and not in CubicCongestionControlOnEcn (where it could be done without the if-check)?